nixos/modules/services/scheduling/scx.nix

```
@@ -101,14 +101,15 @@ in
 
       serviceConfig = {
         Type = "simple";
-        ExecStart = utils.escapeSystemdExecArgs (
-          [
-            (lib.getExe' cfg.package cfg.scheduler)
-          ]
-          ++ cfg.extraArgs
-        );
+        ExecStart = ''
+          ${pkgs.runtimeShell} -c 'exec ${cfg.package}/bin/''${SCX_SCHEDULER_OVERRIDE:-$SCX_SCHEDULER} ''${SCX_FLAGS_OVERRIDE:-$SCX_FLAGS}'
```

```suggestion
          ${pkgs.bash} -c 'exec ${cfg.package}/bin/''${SCX_SCHEDULER_OVERRIDE:-$SCX_SCHEDULER} ''${SCX_FLAGS_OVERRIDE:-$SCX_FLAGS}'
```

I would just use bash for consistency and reproducibility of issues.

---

For some reason I thought this would change to "user shell" which could have some issues with shell expansion (like if zsh is used).